features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-manage/js/api-manage.js

```diff
@@ -13,6 +13,22 @@ function validate_tiers(){
     return false;
 }
 
+function validate_subscription(){
```

Comment:
Fixed in #1494(https://github.com/wso2/carbon-apimgt/commit/6a4f74a2d079f3fbf27a772832e83b12d47adf3e)